http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -379,12 +381,12 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
 
       // Create namespace for nginx
       _ <- logger.info(
-        s"Creating namespace '${config.ingressConfig.namespace.value}' in cluster '${dbCluster.id}' | trace id: ${ctx.traceId}"
+        s"Creating namespace ${config.ingressConfig.namespace.value} in cluster ${dbCluster.id} | trace id: ${ctx.traceId}"
       )
       _ <- kubeService.createNamespace(dbCluster.getGkeClusterId, KubernetesNamespace(config.ingressConfig.namespace))
 
       _ <- logger.info(
-        s"Installing ingress helm chart '${config.ingressConfig.chart}' in cluster '${dbCluster.id}' | trace id: ${ctx.traceId}"
+        s"Installing ingress helm chart ${config.ingressConfig.chart} in cluster ${dbCluster.id} | trace id: ${ctx.traceId}"
```

Comment:
Nit: Values read better to me in logs (i.e. `chart.value` instead of `chart`, and similarly for `id`)

Reply:
+1 me too - will add `.value`. 

Also maybe it's just me but I really don't like quoted string in logs. Not sure why.

Reply:
Single quotes in log messages allow my brain to pause, like with `,` and easily identify that it's a value and not a regular word in the sentence. That being said, it's a personal preference and I don't expect it to work the same way for others 😊